tests/inference/test_query_single.py

```diff
@@ -0,0 +1,77 @@
+import subprocess
+import pytest
+
+# Config matrix
+# models_array = ["gpt2", "gpt2 gpt-j-6b", "gpt2 bloom-560m", "falcon-7b"]
+# model_endpoint_array = ["http://127.0.0.1:8000", None]
+# streaming_response_array = [True, False]
+# max_new_tokens_array = [10, None]
+# temperature_array = [0.7, None]
+# top_p_array = [0.6, None]
+# top_k_array = [5, None]
+
+# Temp Config matrix
+models_array = ["gpt2"]
```

Comment:
could we test all config matrix above and report what don't work?

Reply:
It will take too long for all combos to run for every commit, so I will add another one to nightly discussed with jiafu before.

Reply:
> It will take too long for all combos to run for every commit, so I will add another one to nightly discussed with jiafu before.

In that case, pls write a comment.